environments/go/builder/build.sh

```diff
@@ -4,6 +4,16 @@ if [ -d ${SRC_PKG} ]
 then
     echo "Building in directory ${SRC_PKG}"
     cd ${SRC_PKG}
+
+    if [ -f glide.yaml ]; then
+        go get -u github.com/Masterminds/glide
```

Comment:
Can we download glide in the Dockerfile itself, that way the build doesnt have to wait for it?

Reply:
I can change this, yes :)

The idea was not to bloat the image with all possible vendoring tools but only download on demand (builders are reused, right?)